index.html

```diff
@@ -13,7 +13,12 @@
         <div class="content">
             <nav class="header">
                 <a class="logo" href="#">My Logo</a>
-                <a href="#" class="menu"><img src="img/menu.png" alt=""></a>
+                <a href="#" id="menu"><img src="img/menu.png" alt=""></a>
+                <ul class="nav-links">
+                    <li class="nav-link"><a href="#">Portfolio</a></li>
+                    <li class="nav-link"><a href="#">About</a></li>
+                    <li class="nav-link"><a href="#">Contact</a></li>
+                </ul>
             </nav>
         </div>
```

Comment:
You did well with the desktop version of the portfolio design, however kindly remove the ash line showing underneath your `nav` items of the portfolio, as it is always advisable to stick to the Figma template given as much as possible 😄

Figma design : 
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/3323d867-6358-42e8-a5ff-754cb425664c)

Your design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/a159f497-77cd-425b-807b-05ecbbb543a9)